client/src/app/pages/create-job/create-job.component.html

```diff
@@ -1 +1,91 @@
-<p>create-job works!</p>
+<div class="py-5 flex flex-col items-center justify-center mb-16">
+    <h1 class="sm:text-3xl text-2xl font-medium title-font mb-4 text-gray-900">
+        Create a job listing
+    </h1>
+    <div class="max-w-md w-full bg-white p-8 rounded-lg shadow-md">
+        <form [formGroup]="form" (ngSubmit)="onSubmit()">
+            <div class="mb-6">
+                <label for="title" class="block text-gray-700 text-sm font-bold mb-2">
+                    Position:
+                </label>
+                <textarea id="title" formControlName="position" name="title" rows="1"
+                    class="w-full border-2 rounded-md px-4 py-2 leading-5 transition duration-150 ease-in-out sm:text-sm sm:leading-5 resize-none focus:outline-none focus:border-blue-500"
+                    placeholder="Enter position title here"></textarea>
+            </div>
+
+            <div class="mb-6">
+                <label for="locationType" class="block text-gray-700 text-sm font-bold mb-2">
+                    Loation Type:
+                </label>
+                <select id="location-type" name="locationType" formControlName="locationType"
+                    class="w-full bg-white border-2 rounded-md px-4 py-2 leading-5 transition duration-150 ease-in-out sm:text-sm sm:leading-5 resize-none focus:outline-none focus:border-blue-500">
+                    <option>On-site</option>
```

Comment:
For options, can you specifically use the `JobLocationType` enum to avoid any potential errors?